<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>430</width>
     <height>580</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Начало</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Конец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Профессия</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Педагогическая</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>979</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Даты записываются в формате дд.мм.гггг, с нулями перед числом, если оно однозначное</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>384</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* — обязательно для заполнения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>215</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(оставьте пустым, если он не завершён)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fio_lineedit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>390</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Поступил в учреждение</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="entered_lineedit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>340</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calendar_btn_entered">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>107</y>
     <width>40</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Начало интервала*</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="start_lineedit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>150</y>
     <width>340</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calendar_btn_start">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>147</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string> Конец интервала         </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_lineedit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>190</y>
     <width>340</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calendar_btn_end">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>187</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>142</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Должность*</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="profession_lineedit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>390</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>184</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Должность педагогическая?    </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="is_pedagogical_checkbox">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>280</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="add_row_btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_row_btn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>320</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>562</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ок</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>562</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
